### PR TITLE
Stub out basic UI

### LINKED ISSUE #53: Integrate simple UI into application
**As a** User
**I need** a user
**So that** I can visually access the API

**Details & Assumptions:** 
* Follow example on https://github.com/nyu-devops/lab-flask-bdd
* Frontend files are at: https://github.com/nyu-devops/lab-flask-bdd/tree/master/service/static
* Comment out code in  https://github.com/nyu-devops/lab-flask-bdd/blob/master/service/static/js/rest_api.js for now
* Frontend files should be packaged and deployed with app to IBM cloud
* UI does not need to fully implement functionality to hit various endpoints. These will be done in individual tickets

**Acceptance Criteria:**

```
Given That application is deployed
When I visit the application URL
Then UI should be rendered
```

### PR DESCRIPTION
**Pull request by zhengqixi**
Closes #53

### CODE REVIEW

File: service/static/js/rest_api.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        $("#supplier_id").val("");
+        clear_form_data()
+    });
+
+    // ****************************************
+    // Search for a Supplier 
+    // ****************************************
+
+    $("#search-btn").click(function () {
+
```
Comment (HughHuang96824) [inline service/static/js/rest_api.js:154]: This does not enable search by arbitrary attributes

File: service/static/js/rest_api.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
+$(function () {
+
+    // ****************************************
+    //  U T I L I T Y   F U N C T I O N S
+    // ****************************************
+
+    // Updates the form with data from the response
+    function update_form_data(res) {
```
Comment (HughHuang96824) [inline service/static/js/rest_api.js:8]: I think we have more fields to update (email, address, products, ...)

File: service/static/js/rest_api.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+        $("#supplier_id").val(res._id);
+        $("#supplier_name").val(res.name);
+    }
+
+    /// Clears all form fields
+    function clear_form_data() {
```
Comment (HughHuang96824) [inline service/static/js/rest_api.js:14]: More fields described in the last comment and no category fields

File: service/static/js/rest_api.js [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+    }
+
+    // ****************************************
+    // Create a Supplier
+    // ****************************************
+
+    $("#create-btn").click(function () {
+
+        let name = $("#supplier_name").val();
+
```
Comment (HughHuang96824) [inline service/static/js/rest_api.js:33]: More fields needed, same for the rest function below

File: service/routes.py [hunk 5]
```diff
@@ -25,8 +25,7 @@
 def index() -> Tuple[Response, int]:
```
Comment (HughHuang96824) [inline service/routes.py:25]: Shall we move the endpoint to /api?

**Review by HughHuang96824 — CHANGES_REQUESTED** [review]

File: service/static/js/rest_api.js (hunk 1 above)
Comment (zhengqixi) [reply service/static/js/rest_api.js:154]: I thought we were actually doing the functionality in follow up tickets? This is just to stub out the basic framework.

File: service/static/js/rest_api.js (hunk 2 above)
Comment (zhengqixi) [reply service/static/js/rest_api.js:8]: See above

File: service/static/js/rest_api.js (hunk 3 above)
Comment (zhengqixi) [reply service/static/js/rest_api.js:14]: Will remove irrelevant fields

File: service/static/js/rest_api.js (hunk 4 above)
Comment (zhengqixi) [reply service/static/js/rest_api.js:33]: See above

File: service/routes.py (hunk 5 above)
Comment (zhengqixi) [reply service/routes.py:25]: Not opposed. But do you know if this is listed as a requirement or anything anywhere? I don't recall having things at `api/suppliers`.

Comment (HughHuang96824) [reply service/routes.py:25]: In the slack homework channel, someone asked how the UI is connected to the service on Nov 13th. You can check out the conversation below.

**Review by HughHuang96824 — APPROVED** [review]

File: service/routes.py (hunk 5 above)
Comment (zhengqixi) [reply service/routes.py:25]: Sure I can make this change here. Think FlaskRestX will do some of this work down the line, so we'll need to be careful else we end up with `api/api/`

**Pull request merged**